Support for ActorPacks

cybersec-code opened the pull request:
An initial set of features to support ActorPack validation and insertion into the database. A new taxonomy was introduced (country) as well as new tables to the schema.

soad003 commented on tagpack/db/tagstore_schema.sql:
```diff
@@ -80,6 +80,39 @@ CREATE INDEX tag_label_index ON tag (label);
+CREATE TABLE actor_jurisdictions (
+    id                  SERIAL      PRIMARY KEY,
+    actor_id            VARCHAR     REFERENCES actor(id) ON DELETE CASCADE,
+    country_id          VARCHAR     REFERENCES concept(id) ON DELETE CASCADE
```
is this constraint to the concepts table intended?

soad003 replied:
Maybe i misunderstand the schema. Have seen that the tests also treat coutries as concepts.

soad003 commented on tagpack/tagstore.py:
```diff
@@ -227,7 +332,7 @@ def get_quality_measures(self, currency="") -> float:
         return {keys[i]: v for row in self.cursor.fetchall() for i, v in enumerate(row)}
 
-    def calculate_quality_measures(self) -> float:
+    def calculate_quality_measures(self) -> dict:
```
Is this quality view something we want to keep up to date in general? If so maybe add call to this to tp-tool tagstore refresh_views

cybersec-code commented:
Thanks for the feedback,

- I added an [actors file](https://github.com/graphsense/graphsense-tagpacks/blob/f4f9a0558448eb90ff42a9d8c509371494cb0980/actors/demo.yaml) to the tagpacks repo for testing. We will add more ActorPacks to this folder in the near future.
- We are still defining how to integrate actors with tags, so there are no tests for it at the moment.
- We added country as a taxonomy, that's why the constraint references `concept(id)` in the schema.
- `tagpack-tool taxonomy insert country` is working now.
- I think for now it's not necessary to keep up to date the quality view, because we want to take "snapshots" of those results at different points in time. But maybe in the future this changes.
- I'm updating the README file

soad003 commented:
Thanks for the clarification and updates. 

I have tested what I can test so far (import of actors, computing quality stats etc.) and quickly skimmed over the source-code.

I think its got to go like.

soad003 reviewed (approved):
I have integrated recent changes from develop. I think it all looks fine. Will merge into develop now. 

@cybersec-code: there have been some changes regarding folder structure of the repo and some others infrastructure change like enforced formatting and linting via pre-commit hooks. If you have any questions or troubles while developing, let me know. 

Thanks everyone for the effort.
